src/pages/personalisation/BaseQuestion.js

```diff
@@ -172,6 +172,11 @@ class BaseQuestion extends React.Component {
     static getSearch(request: iss.searchRequest): ?iss.searchRequest {
         let value = this.answer;
 
+        if (value == "(skipped)") {
```

Comment:
It's got to round-trip safely through JSON and also render nicely with `toString()` - so it might be difficult.